src/backend/routers/paras.ts

```diff
@@ -34,13 +35,23 @@ export const paraProcedures = {
     )
     .mutation(async (req) => {
       const { first_name, last_name, email, role } = req.input;
+      // In the SCHEMA, email has the "unique" constraint TEXT UNIQUE NOT NULL, which will prevent duplicate paras from being inputted
+
+      // this is where Nodemailer will send an email to the email address of the para inputted by the cm
+      await transporter.sendMail({
```

Comment:
might make more sense to send the email after inserting to our database, don't want to send an email if the insert fails for some reason

Reply:
Hi Max, in paras.ts, I moved transporter.sendMail to after .execute. I'm not sure if that is what you meant. Please let me know.

Reply:
looks good!